Content.Shared/CriminalRecords/Components/CriminalRecordsConsoleComponent.cs

```diff
@@ -1,15 +1,18 @@
 using Content.Shared.CriminalRecords.Systems;
+using Content.Shared.IdentityManagement;
 using Content.Shared.Radio;
+using Content.Shared.Security;
 using Content.Shared.StationRecords;
+using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
 
 namespace Content.Shared.CriminalRecords.Components;
 
 /// <summary>
 /// A component for Criminal Record Console storing an active station record key and a currently applied filter
 /// </summary>
-[RegisterComponent]
-[Access(typeof(SharedCriminalRecordsConsoleSystem))]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[Access(typeof(SharedCriminalRecordsConsoleSystem), typeof(SharedIdentitySystem))]
```

Comment:
add public api to the shared console system instead of this

also i think it should be stored on the station since there can be multiple consoles

Reply:
I don't store the names separately anymore, but instead directly check all existing criminal records if the name of the new identity has a criminal status attached to it.